### ISSUE TITLE
Fix misaligned Attempt and Response entities descriptive text

### ISSUE THREAD

**Issue by arwhyte** [IMSGlobal/caliper-spec#393]
A concern was expressed that Survey Profile `Response` sub types inherit an `attempt<Attempt>` property that leaks assignable concepts and terms into the profile.

A re-reading of the Caliper 1.1 spec suggests that this concern is based on some poorly worded descriptive text in the 1.1 spec that is not aligned with the actual type values of the property in question.

### Proposal
We make no changes to the Survey Profile.  All current Survey response sub types of `Response` remain sub types of `Response`.  We can do this because `Attempt.assignable` is not typed narrowly as `AssignableDigitalResource` but as `DigitalResource` both in the 1.1 spec and in the 1.1 ontology.  Therefore the optional `Response.attempt<Attempt>` does not inject, so to speak, an assignable entity constraint into the Survey Profile.  

There are two textual changes that do need to be made.  

The 1.1 description of `Attempt` (https://github.com/IMSGlobal/caliper-spec/blob/master/caliper-spec.md#attempt) does not align with the actual type definition: `Attempt.assignable<DigitalResource>`.  It says:  "A Caliper `Attempt` provides a count of the number of times an `actor` has interacted with an `AssignableDigitalResource` along with start time, end time and duration information. An `Attempt` is generated as the result of an action such as starting an `Assessment`."  It should actually say: "... has interacted with a `DigitalResource` . . . .".  We could also tweek the `Attempt.assignable` description without harm from "The `DigitalResource` that constitutes the object of the assignment." to "The `DigitalResource` that constitutes the object of the attempt."

Likewise the 1.1 description of `Response` (https://github.com/IMSGlobal/caliper-spec/blob/master/caliper-spec.md#c37-response) needs to be tweeked: "A Caliper `Response` is a generic type that represents the selected option generated by a `Person` interacting with an `AssessmentItem`."  It should actually say: "interacting with a `DigitalResource`, typically an `AssessmentItem` or other `AssignableDigitalResource` . . ." or words to that effect.  

In short, the current 1.1 descriptions of both `Attempt` and `Response` misrepresent the actual type constraint imposed on `Attempt.assignable` which is rather more relaxed than we all think it is.  We could edit both of these entity descriptions under the "errata" rules for 1.1 and/or simply update the descriptions for 1.2 in the develop branch.
